js/ui/map.js

```diff
@@ -287,6 +287,27 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
     },
 
     /**
+     * Set constraint on the map's geographical bounds.
```

Comment:
Can this be more specific? Like, "Pan or zoom operations that would result in displaying regions that fall outside of the bounds instead _____."